everyvoice/tests/regression/prep-datasets.sh

```diff
@@ -55,13 +57,20 @@ cat <<'==EOF==' > "$dir"/test.txt
 තතො නං දුක්‌ඛමන්‌වෙති, චක්‌කංව වහතො පදං.
 ==EOF==
 echo "අක-ෂර" > "$dir"/test2.txt
+done
```

Comment:
Except for the four lines of text and the line with just `==EOF==`, the code above this `done` line should be indented to match the loop it's in. It took me a moment to figure out why there was a `done` here with the lines above not indented I didn't immediately see it was ending the loop.